kplclient_test.go

```diff
@@ -25,3 +25,33 @@ func TestHappy(t *testing.T) {
 	//wait before shutting down
 	time.Sleep(20 * time.Second)
 }
+
+func TestNegative(t *testing.T) {
+	//Enabled it when testing for negative scenario. Pass invalid credential to AWS
+	t.SkipNow()
+	//give the test harness a chance to boot
+	time.Sleep(10 * time.Second)
```

Comment:
Yikes, this shouldn't be necessary. ??

Reply:
This is to make sure kplserver has started before client establish the connection. We should probably do retry in code.

Reply:
See below